### ISSUE TITLE
About package install error

### ISSUE THREAD

**Issue by YangJae96** [wx-zhang/spu#1]
Hello.
Thank you for your great work!

When I tried to download  cosine-annealing-warmup==2.0, I get below error.

ERROR: Could not find a version that satisfies the requirement cosine-annealing-warmup==2.0 (from versions: none)
ERROR: No matching distribution found for cosine-annealing-warmup==2.0

Also, I get below error when I used "conda env create -f environment.yml" command.

Pip subprocess error:
ERROR: Ignored the following versions that require a different python version: 1.2.0 Requires-Python >=3.9; 1.2.1 Requires-Python >=3.9; 1.2.1rc1 Requires-Python >=3.9
ERROR: Could not find a version that satisfies the requirement cosine-annealing-warmup==2.0 (from versions: none)
ERROR: No matching distribution found for cosine-annealing-warmup==2.0

**Comment by wx-zhang** [comment]
Hi @YangJae96, 

Thank you for your comment!
You could follow this page to install the learning rate scheduler
[https://github.com/katsura-jp/pytorch-cosine-annealing-with-warmup](url)

**Comment by YangJae96** [comment]
Hi.

I cannot run the command that you metioned!
python main.py dataset=[cifar100 | cub | cars | aircraft | gtsrb | birdsnap ]

Following error is given.

![image](https://github.com/wx-zhang/spu/assets/57695621/e8eff936-f425-41a2-ba6a-3bfc0fde5139)

Could I get some help?!

**Comment by wx-zhang** [comment]
Hi @YangJae96 ,

This means you may select one dataset from these options.
